docker-images/baleine/Dockerfile

```diff
@@ -1,5 +1,28 @@
 FROM ubuntu:latest
 
+# Install necessary binaries for tutorials
+RUN apt update -y && apt upgrade -y && apt install kmod iw iputils-ping openssh-client openssh-server bash netcat iproute2 net-tools tcpdump dnsutils netcat nano vim -y
+
+# Setup Bash prompt to not get lost when in interactive mode
+RUN echo "PS1='[CONTAINER] ${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '" | tee /root/.bashrc > /dev/null
+
+#Setting up OpenSSH server
+RUN sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config
+RUN sed -i 's/#Port 22/Port 2222/g' /etc/ssh/sshd_config
+RUN sed -i 's/#ListenAddress 0.0.0.0/ListenAddress 0.0.0.0/g' /etc/ssh/sshd_config
+RUN echo "auth sufficient pam_permit.so" | tee -a /etc/pam.d/sshd >> /dev/null
```

Comment:
I'm surprised we need to mess with the pam config, given that it's explicitly turned off for ssh, or what am I missing ?

Reply:
The change on PAM is made so that we don't need a password to login when using SSH.

Reply:
line 11 suggests ssh will not use PAM at all; why configure it then ?
(and while we're at it, maybe it would be more explicit to set `UsePAM no` so there's no doubt about the default behaviour)